test/deploy.py

```diff
@@ -0,0 +1,486 @@
+#!/usr/bin/python3
+import os
+import sys
+import pwd
+import json
+import yaml
+import time
+import pylxd
+import string
+import random
+import subprocess
+
```

Comment:
Minor style nit: suggest running the module through black for automatic formatting

Reply:
ack